src/server.js

```diff
@@ -0,0 +1,56 @@
+const express = require('express');
+const bodyParser = require('body-parser');
+
+const jsonParser = bodyParser.json({ limit: '32mb' });
+const PORT = 5100;
+
+const app = express();
+const containers = require('./containers');
+
+const upgrade = async (req, res) => {
```

Comment:
The API is structured like this because the same request will be sent to the upgrade service for medic-hosted, which requires a different payload (pairs of service,image tags). 
When we discussed this, we agreed that API will make one request, regardless of the upgrade service used.

Reply:
So, even if we would just do one file, we would still need the body parser, because of how the request will be built (tje medic hosted upgrade service requires the payload to be json).

Reply:
Ok... It's not worth changing everything around now (even if we wanted to) so let's go with what you have.